src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+import "./Home.css";
+import bottleOne from "../images/bottle-one.png";
+import bottleTwo from "../images/bottle-two.png";
+
+export default function Home() {
+  return (
+    <div className="home">
+      <div className="headline-wrapper">
+        <h1 className="headline">routinely</h1>
+      </div>
+      <div className="bottle-wrapper-one">
+        <a href="/products" className="product-link">
```

Comment:
If you are using `react-router-dom` you should be using either the `Link` component or the `NavLink` component instead of the `a` element:
[Link](https://reactrouter.com/web/api/Link)
[NavLink](https://reactrouter.com/web/api/NavLink)